front/types/chat.ts

```diff
@@ -1,3 +1,9 @@
+export type ChatTimeRange = {
```

Comment:
This type is specific to the timerange component for now so let's have it there instead of here :+1: and import it in the Chat page from the time range component file

`front/types` include mostly (exclusively really) types that go through the network boundary